src/OpenTelemetry.Exporter.ZPages/ZPagesExporter.cs

```diff
@@ -26,16 +28,27 @@ namespace OpenTelemetry.Exporter.ZPages
     /// </summary>
     public class ZPagesExporter : SpanExporter
     {
+        internal readonly ZPagesExporterOptions Options;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ZPagesExporter"/> class.
+        /// </summary>
+        /// <param name="options">Options for the exporter.</param>
+        public ZPagesExporter(ZPagesExporterOptions options)
+        {
+            this.Options = options;
+        }
+
         /// <inheritdoc />
         public override Task<ExportResult> ExportAsync(IEnumerable<SpanData> batch, CancellationToken cancellationToken)
         {
-            throw new NotImplementedException();
+            return (Task<ExportResult>)Task.CompletedTask;
```

Comment:
You could return like this instead of doing inline cast:

```suggestion
            return Task.FromResult(ExportResult.Success);
```